File: a1-programming-basics-javascript/02b-conditional-statements-exercise/07-shopping.js
function computerShopping(input) {
  let budget = Number(input[0]);
  let amountVGA = Number(input[1]);
  let amountCPU = Number(input[2]);
  let amountRAM = Number(input[3]);

  priceVGA = amountVGA * 250;
  priceCPU = priceVGA * 0.35 * amountCPU;
  priceRAM = priceVGA * 0.1 * amountRAM;
  totalPrice = priceVGA + priceCPU + priceRAM;

  if (amountVGA > amountCPU) {
    discount = totalPrice * 0.15;
    totalPrice = totalPrice - discount;
  }

  absMoney = Math.abs(budget - totalPrice).toFixed(2);

  if (budget > totalPrice) {
    console.log(`You have ${absMoney} leva left!`);
  } else {
    console.log(`Not enough money! You need ${absMoney} leva more!`);
  }
}

File: b1-programming-basics-python/02b-conditional-statements-exercise/07_shopping.py
budget = float(input())
num_vga = int(input())
num_cpu = int(input())
num_ram = int(input())

price_vga = num_vga * 250
price_cpu = num_cpu * price_vga * 0.35
price_ram = num_ram * price_vga * 0.1

total_price = price_vga + price_cpu + price_ram

if num_vga > num_cpu:
    total_price *= 0.85

if budget < total_price:
    print(f"Not enough money! You need {total_price - budget:.02f} leva more!")

else:
    print(f"You have {budget - total_price:.02f} leva left!")
